### PR TITLE
Cypress test. Error сannot read property at saving job.

### PR DESCRIPTION
**Pull request by DmitriyOparin**
Cypress test. Error сannot read property at saving job.

### Checklist

- [X] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [X] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [X] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_tasks_objects/pr_2203_error_сannot_read_property_at_saving_job.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            cy.changeWorkspace('Attribute annotation', labelName);
+        });
+
+        it('Save job and go to previous frame at saving job', () => {
+            cy.saveJob();
+            cy.get('body').type('d');
+        });
+
+        it('Page with the error is missing', () => {
+            cy.wait(100);
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/pr_2203_error_сannot_read_property_at_saving_job.js:51]: Waiting for a specific time is not the best solution here. Probably need to wait till saving modal window disappears.
This time depends on factor that we cannot control: client-server bandwidth.

**Review by bsekachev — CHANGES_REQUESTED** [review]

File: tests/cypress/integration/actions_tasks_objects/pr_2203_error_сannot_read_property_at_saving_job.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            cy.contains('Oops, something went wrong').should('not.exist');
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/pr_2203_error_сannot_read_property_at_saving_job.js:52]: Please find the element by class: ``cvat-global-boundary``, not by text

File: tests/cypress/integration/actions_tasks_objects/pr_2203_error_сannot_read_property_at_saving_job.js (hunk 1 above)
Comment (DmitriyOparin) [reply tests/cypress/integration/actions_tasks_objects/pr_2203_error_сannot_read_property_at_saving_job.js]: Thanks! Function cy.wait replaced to wait XHR request success saving.

File: tests/cypress/integration/actions_tasks_objects/pr_2203_error_сannot_read_property_at_saving_job.js (hunk 2 above)
Comment (DmitriyOparin) [reply tests/cypress/integration/actions_tasks_objects/pr_2203_error_сannot_read_property_at_saving_job.js]: Locator changed for searching element.

**Comment by dvkruchinin** [comment]
@DmitriyOparin 
Please adapt the test to the updated object creation functionality.

**Comment by DmitriyOparin** [comment]
> @DmitriyOparin
> Please adapt the test to the updated object creation functionality.

**Comment by DmitriyOparin** [comment]
Test is adapt for new develop branch

**Review by bsekachev — APPROVED** [review]

**Pull request merged**